src/commonMain/kotlin/org/kosat/heuristics/VariableSelector.kt

```diff
@@ -101,7 +109,7 @@ class VSIDS(private var numberOfVariables: Int = 0) : VariableSelector() {
                 break
             }
         }
-        return v
+        return v * 2
```

Comment:
Why?
Are you trying to convert `v` to a "literal" (positive by default)? In this case, your function name is quite wrong: you promised "`get...Variable`", but returning literal.
You already know the solution: wrap all variables and literals with `Var` and `Lit` inline-classes in order to eliminate the obscurity of `Int`s.

If you are doing `2*v` for any other reason, then please explain!